eng/azure-sdk-deps.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Should this pick up the `$(RequiredTargetFrameworks)` property, or whatever? Less duplication to manage, though I can't imagine we'll stray from "netstandard2.0" anytime soon.